storage-node/src/cache/policy/mod.rs

```diff
@@ -50,13 +50,21 @@ impl DataStoreCacheValue for ParpulseDataStoreCacheValue {
 ///
 /// There are different cache policies for the data store cache, such as LRU, LRU-K, etc. See
 /// other files in this module for more details.
-pub trait DataStoreCache {
+pub trait DataStoreReplacer {
```

Comment:
```suggestion
pub trait DataStoreReplacer : Send + Sync {
```
so no need to `R: DataStoreReplacer + Send + Sync` everywhere else?